project/ScalafixBuild.scala

```diff
@@ -177,6 +177,7 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
   )
 
   private val PreviousScalaVersion: Map[String, String] = Map(
+    "2.13.8" -> "2.13.7"
```

Comment:
help MiMa find the previous artifact (there is so and won't be any 2.13.8 artifact for the current version of scalafix modules)